tests/sentry/test_wsgi.py

```diff
@@ -0,0 +1,59 @@
+import os
+import select
+import subprocess
+import sys
+import time
+
+from sentry.services.http import PYUWSGI_PROG
+
+
+def test_wsgi_init():
+    os.environ["UWSGI_HTTP_SOCKET"] = "127.0.0.1:9001"
+    os.environ["UWSGI_MODULE"] = "sentry.wsgi:application"
+    os.environ["TEST_WARMUP"] = "1"
+    cmd = (
+        sys.executable,
+        "-c",
+        PYUWSGI_PROG,
+    )
```

Comment:
not really what I had in mind -- instead just run a subprocess which imports `sentry.wsgi` and then validate whatever assertions you want afterwards (url modules in `sys.modules`, url cache populated, etc.)

should be much much simpler and won't need to do any busy waiting

Reply:
yeah much simpler! just changed it.